### DISCUSSION TITLE
Integrating IXP Granularity with our Global Report

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#27] [General]
@romain-fontugne This feature is related to [Adding Internet Exchange Point (IXP) Names in Network Alarms API Endpoint](https://github.com/orgs/InternetHealthReport/discussions/26). In order to implement this feature we need to have a dataset either from IHR or other external data sources where we can get peering relationships between ASNs and IXPs. I'd love also to know what you think about this feature ?

**Comment by romain-fontugne** [comment]
This is an interesting feature, it could add a bit of complexity. We can get ASN/IXP relationship from [PeeringDB](https://www.peeringdb.com/) or from [IYP](https://iyp.iijlab.net/) which is going to be soon queryable from IHR code base.

But also I want to mention that IXPs are visible only in the network delay alarms. The other alarms will never show IXPs. So I think it is a feature that would be nice to have, but I'd put it a lower priority

**Reply by moawnallah** [reply]
Thanks that makes sense!
